web/api/v1/api.go

```diff
@@ -436,6 +440,17 @@ func (api *API) alertmanagers(r *http.Request) (interface{}, *apiError) {
 	return ams, nil
 }
 
+type prometheusConfig struct {
+	RawFormat string `json:"rawFormat"`
```

Comment:
I wonder if rawFormat is the best name here. I might be wrong, but isn't this the representation without comments, secrets and reformatted by our printer?